api/metrics/rent_burdened.py

```diff
@@ -0,0 +1,82 @@
+from api.utils.database import rows_to_dicts
+
+
+class RentBurdenedMetrics:
+    """
+    Metrics for rent burdened households data.
+    """
+
+    def __init__(self, con):
+        self.con = con
+
+    def rent_burdened(self, year, segment):
+        """
+        Returns the percent of households who are rent burdened in the area,
+        and the area number.
+        Args:
+            year (int): period ending year to filter by
+            segment (str): population segment to filter by
+        """
+        query = """
+        SELECT
+            area_number,
+            CAST(value AS INTEGER) AS value
```

Comment:
If your metric is for a percentage or rate, return the value as a value between 0 and 1. We will take care of formatting it as a percentage on the frontend.

If all of the metrics that are rates or percentages return values between 0 and 1 it makes it easier to display color scales and to compare different metrics on the same axis when making data visualizations.

Reply:
Should I just add an sql script to divide my values by 100 before I load them into the database?
Referring to` pipeline/load/rent_burdened_households.sql`

Reply:
It's okay to do the division here in the metric! That's what the other metrics do.

But if you want to change what's in the database, I would do the division in your transform Python script. The load SQL is just for defining the table schema, not modifying the data.